mm2src/mm2_rpc/src/data/legacy.rs

```diff
@@ -254,4 +256,95 @@ pub struct MmVersionResponse {
     pub datetime: String,
 }
 
-fn get_true() -> bool { true }
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(tag = "method", rename_all = "lowercase")]
+pub enum ActivationRequest {
```

Comment:
`ActivationRequest`, `EnableRequest`, `ElectrumRequest` and `Server` are only used in `adex-cli`, you should probably move them there unless you plan to use them in mm2 later.